src/lib/audienceChannel/styles/agenda.css

```diff
@@ -0,0 +1,13 @@
+#home {
```

Comment:
home is a common element for all pages in audience channel, can we put it into a common css?